src/components/backdrop/backdrop.scss

```diff
@@ -1,6 +1,7 @@
 md-backdrop {
   z-index: $z-index-backdrop;
   &.md-menu-backdrop {
+    position: fixed !important;
```

Comment:
Can you add a comment explaining the `!important` ? 

(always writing an explanation for every use of `!important` has been a personal policy that has served me well)